web/Views/Landlord/Register.cshtml

```diff
@@ -8,96 +8,97 @@
 
 <p>Tell us a bit about yourself</p>
 <hr/>
-
-<div class="row">
-    <div class="col-md-4">
-        <form method="post" asp-controller="Landlord" asp-action="RegisterPost" id="landlordEntry">
-        <div asp-validation-summary="All" class="text-danger"></div>
-            <div class="registerTab" id="Name">
-                <div class="form-group my-3">
-                    <label>Title:</label>
-                    @Html.DropDownList("Title", new List<SelectListItem>
-                    {
-                        new SelectListItem { Text = "Mr", Value = "Mr", Selected = true },
-                        new SelectListItem { Text = "Mrs", Value = "Mrs" },
-                        new SelectListItem { Text = "Miss", Value = "Miss" },
-                        new SelectListItem { Text = "Ms", Value = "Ms" },
-                        new SelectListItem { Text = "Dr", Value = "Dr" }
-                    })
+<body onload="showTab(0)">

```

Comment:
There's already a `<body>` tag in `_Layout.cshtml` that we're inside of here. I believe the `onload` attribute also works on other elements, can you try just adding it to the existing `<div>` or did that not work?

Reply:
I will keep trying to hunt around to see if there is a nice fix for a little while